### PR TITLE
Ingested MT Summit 2005.

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/05/MTS-2005-TOC.htm for original.

### CODE REVIEW

**Comment by mjpost** [comment]
Missing PDFs, then I'll review.

**Comment by davidstap** [comment]
Thanks for adding the bibkeys. The PDFs are uploaded now.

**Comment by mjpost** [comment]
Thanks! I wanted to test out the bib key script.

This is missing the tutorial notes. We should be sure to get this, since it has an early tutorial from Franz Och's time at Google.

File: data/xml/2005.mtsummit.xml [hunk 1]
```diff
@@ -0,0 +1,1128 @@
...
+      <author><first>Kazunari</first><last>Ishida</last></author>
+      <author><first>Nobuhiko</first><last>Shimizu</last></author>
+      <author><first>Toshizumi</first><last>Ohta</last></author>
+      <pages>29-34</pages>
+      <url hash="9056e721">2005.mtsummit-wpt.5</url>
+      <abstract>Large-scale parallel corpus is extremely important for translation memory, example-based machine translation, and the support system to create English sentences. Organized collection or establishment of large-scale corpus is currently ongoing; however it is a difficult project in terms of copyrights as well as economic efficiency. To investigate general tendency of large-scale corpus helps to improve economical efficiency of parallel corpus collection as well as system establishment. In this study, therefore, the relationship between the scale of parallel corpus and the degree of correspondence is clarified, using parallel corpus for patents.</abstract>
+      <bibkey>okada-etal-2005-quality</bibkey>
+    </paper>
+    <paper id="6">
+      <title>Less, Easier and Quicker in Language Acquisition for Patent <fixed-case>MT</fixed-case></title>
```
Comment (mjpost) [inline data/xml/2005.mtsummit.xml:1039]: "Less, Easier, and Quicker" (quotes in the original)

File: data/xml/2005.mtsummit.xml [hunk 2]
```diff
@@ -0,0 +1,1128 @@
...
+      <pages>471-475</pages>
+      <url hash="d4173279">2005.mtsummit-posters.22</url>
+      <abstract>This paper describes a cellular-telephone-based text-to-text translation system developed at Transclick, Inc. The application translates messages bi-directionally in English, French, German, Italian, Spanish and Portuguese. This paper describes design features uniquely suited to hand-held-device based translation systems. In particular, we discuss some of the usability conditions unique to this type of application and present strategies for overcoming usability obstacles encountered in the design phase of the product.</abstract>
+      <bibkey>barrett-levin-2005-usability</bibkey>
+    </paper>
+  </volume>
+  <volume id="osmtw" ingest-date="2021-07-23">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit X</booktitle>
+      <publisher>?? Not mentionned on TOC</publisher>
```
Comment (mjpost) [inline data/xml/2005.mtsummit.xml:761]: This should be deleted

File: data/xml/2005.mtsummit.xml [hunk 3]
```diff
@@ -0,0 +1,1128 @@
...
+      <author><first>Robert</first><last>Levin</last></author>
+      <pages>471-475</pages>
+      <url hash="d4173279">2005.mtsummit-posters.22</url>
+      <abstract>This paper describes a cellular-telephone-based text-to-text translation system developed at Transclick, Inc. The application translates messages bi-directionally in English, French, German, Italian, Spanish and Portuguese. This paper describes design features uniquely suited to hand-held-device based translation systems. In particular, we discuss some of the usability conditions unique to this type of application and present strategies for overcoming usability obstacles encountered in the design phase of the product.</abstract>
+      <bibkey>barrett-levin-2005-usability</bibkey>
+    </paper>
+  </volume>
+  <volume id="osmtw" ingest-date="2021-07-23">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit X</booktitle>
```
Comment (mjpost) [inline data/xml/2005.mtsummit.xml:760]: This should be "Workshop on open-source machine translation"

**Review by mjpost — COMMENTED** [review]
See scattered mistakes. The tutorial volume should probably be called "Proceedings of Machine Translation Summit X: Tutorial notes", and I'd move it up above the workshops.

**Comment by davidstap** [comment]
Not sure how I missed the tutorials, but they are included now.

**Review by mjpost — APPROVED** [review]

**Pull request merged**